test/multi.html

```diff
@@ -43,6 +44,20 @@
     </template>
   </test-fixture>
 
+  <!--
+  NOTE(cdata): Enable test-fixture when polymer/polymer#2495 is resolved
+  -->
+  <!--<test-fixture id="repeatedItems">
```

Comment:
Right now, the test runs. The `test-fixture` was borked, so there is a danger that if we use these elements in other tests, there will be state leak. Also: the problem in the cited issue affects all IE.